Dockerfile.otel-collector

```
@@ -0,0 +1,43 @@
+FROM debian:latest as journal
+RUN apt update
+RUN apt install -y systemd libssl-dev
```

```suggestion
RUN apt update  \
  && apt upgrade -y  \
  && apt install -y systemd libssl-dev
```

RUN apk upgrade --no-cache --no-progress \
  && apk add --no-cache --no-progress libc6-compat \
  && apk del --no-cache --no-progress apk-tools alpine-keys